### ISSUE TITLE
Signup Wizard: Not passing Group Member role onto connection

### ISSUE THREAD

**Issue by jcamarin** [secc/RockPlugins#90]
We recently upgrade this plugin and now our sign-up wizard is not passing the placement group or the placement group configuration into the Connection so that information is lost and we have no idea what the person is signing up for.

**Comment by nathanschindler** [comment]
Have you found a solution to this yet? We are trying to use the signup genius for our Easter signups and noticed the same thing.

Thanks, Nathan

**Comment by jcamarin** [comment]
yes! If I remember correctly was not mapping it in the totals tab only in the intial setup screen.
<img width="1180" alt="Screen Shot 2021-03-16 at 9 59 36 AM" src="https://user-images.githubusercontent.com/44300741/111331054-87d66080-863e-11eb-90aa-547e9833a1fb.png">

**Comment by jcamarin** [comment]
You can see my path to figuring it out here: https://chat.rockrms.com/channel/plugin-discussion?msg=S4gr4hEhXsgccvQs6

**Comment by nathanschindler** [comment]
Thank you for the quick response! I'm going to start reading :)

**Comment by nathanschindler** [comment]
Thank you so much jcamarin. I've got it working! You Made my day!

**Comment by jcamarin** [comment]
AWESOME! I'm so glad and I'm so glad to help! Have a great day!

**Comment by AmandaGeorge** [comment]
@jcamarin looks like you resolved this issue and you could close it now.
